.gitignore

```diff
@@ -23,6 +23,7 @@ pyinstaller/version.txt
 pyinstaller/electron/version-data.json
 pyinstaller/electron/downloadloc.js
 pyinstaller/electron/node_modules
+pyinstaller/electron/package-lock.json
```

Comment:
Shouldn't that file not get checked in?

Reply:
Yes, that's why I'am adding it to `.gitignore` here ...